app/images/icons/README.md

```diff
@@ -0,0 +1,9 @@
+# Icon images
+
+The svg icons in this folder work directly with the `<Icon />` component in `ui/components/component-library/icon/icon.js`.
+
+It works in conjunction with the generate icon names script in `development/generate-icon-names.js` to display all of the icons in this folder.
+
+## IMPORTANT!!
+
+If you are not intending for your svg image to be used with the `<Icon />` component **DO NOT** add it to this folder. For more information read the `<Icon />` component [README.mdx](../../../ui/components/component-library/icon/README.mdx) or checkout the [storybook](https://metamask.github.io/metamask-storybook/index.html?path=/story/ui-components-component-library-icon-icon-stories-js--default-story)
```

Comment:
The github pages version of the icons is currently broken I will fix this in another PR that will be connected to https://github.com/MetaMask/metamask-extension/issues/15974. I just wanted to get this removal of the mistake script in promptly